org.lamport.tla.toolbox.yml

```diff
@@ -0,0 +1,55 @@
+app-id: org.lamport.tla.toolbox
+runtime: org.freedesktop.Platform
+runtime-version: "19.08"
+sdk: org.freedesktop.Sdk
+command: tlatoolbox
+
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=wayland
+  - --socket=pulseaudio
+  - --share=network
+  - --device=dri
+  - --filesystem=home
+
+modules:
+
+  - name: tlatoolbox
+    no-autogen: true
+    sources:
+      # Fetch TLA Toolbox release
+      - type: archive
+        url: https://github.com/tlaplus/tlaplus/releases/download/v1.6.0/TLAToolbox-1.6.0-linux.gtk.x86_64.zip
+        sha256: c1622b4e21fc3985a276e8814478b702c23f20e4c114773f9a5ec9542b50199b
+        x-checker-data:
+          type: html
+          url: https://api.github.com/repos/tlaplus/tlaplus/releases/latest
+          version-pattern: https://github.com/tlaplus/tlaplus/releases/tag/v(\d\.\d\.\d)
+          url-template: 'https://github.com/tlaplus/tlaplus/releases/download/v$version/TLAToolbox-$version-linux.gtk.x86_64.zip'
+      
+      # Desktop file.
+      - type: file
+        path: org.lamport.tla.toolbox.desktop
+
+      # Icon file. Source: https://github.com/tlaplus/tlaplus/tree/master/toolbox/org.lamport.tla.toolbox.feature.branding/rootfiles
```

Comment:
Nitpick: It would be nice to have it included in the zip.

Reply:
The icon file comes from the given url, but a copy is included in the repository (`splash-256x256.png` and `splash-480x480.png`) to avoid broken references.